output_delegation.go

```diff
@@ -147,6 +147,15 @@ func (d *DelegationOutput) VBytes(rentStruct *RentStructure, _ VBytesFunc) VByte
 		d.ImmutableFeatures.VBytes(rentStruct, nil)
 }
 
+func (d *DelegationOutput) WorkScore(workScoreStructure *WorkScoreStructure) WorkScore {
+	return d.Conditions.WorkScore(workScoreStructure) +
+		d.ImmutableFeatures.WorkScore(workScoreStructure) +
+		workScoreStructure.FactorData.Multiply(DelegationIDLength) +
```

Comment:
why not write all in one FactorData.Multiply? That reduces the amount of multiplications.

Reply:
done